### ISSUE TITLE
I cant use tinleaf to install games from pc

### ISSUE THREAD

**Issue by berk-can** [blawar/tinleaf#12]
I've installed tinleaf 1.3.2 and I have atmosphere 0.12.0
I've installed zadig and replaced driver, then I started ns-usbloader selected my game file but whenever I start tinleaf and select install over usb it says No USB connection detected Plug in to a compatible device to install over USB
btw I use same usb cable to send payload while starting device so I dont think there is a problem with my cable
Plese help

**Comment by blawar** [comment]
yes, it works with ns-usbloader.  you should remove all of your switch drivers and reinstall.  also try a different usb port.
